pandas/io/json/_normalize.py

```diff
@@ -26,12 +26,42 @@ def convert_to_line_delimits(s):
     return convert_json_to_lines(s)
 
 
+def _parse_use_keys(use_keys: Optional[Union[str, List[str], Callable]] = None):
```

Comment:
Could you type the return as Callable here? Also I think can simplify annotation by just requiring the argument

Reply:
You mean making use_keys a required argument, and handling `None` case in the calling function?